pkgs/applications/editors/rcptt/default.nix

```diff
@@ -0,0 +1,85 @@
+# Intende useage
+# Build the elexis-congo app via mvn -V clean verify -f elexis-congo/pom.xml
+# nix-build Elexis_Congo.nix
+# start one of the binaries under result/bin/
+#
+{ lib, pkgs, stdenv, bundlerEnv, ruby, makeWrapper, makeDesktopItem
+, gnome, git, gtk3, jdk11, fontconfig, freetype, glib
+, gsettings-desktop-schemas, webkitgtk, docutils, perl, xorg, zlib
+, libXrender, libsecret, unzip, rsync, fetchzip, ... }:
+let
+  desktopItem = makeDesktopItem {
+    name = "RCPTT";
+    exec = "rcptt";
+    icon = "rcptt";
+    comment = "RCP Testing Tool is a project for GUI testing automation of Eclipse-based applications. RCPTT is fully aware about Eclipse Platform's internals, hiding this complexity from end users and allowing QA engineers to create highly reliable UI tests at great pace.";
+    desktopName = "RCPTT";
+    genericName = "RCP Testing Tool";
+    categories = [ "Development" "IDE" "Java"  ];
+  };
+in
+
+stdenv.mkDerivation rec {
+  pname = "RCPTT";
+  version = "2.5.3" ;
+  name = "${pname}-${version}";
+  src = fetchzip {
+    url = "https://download.eclipse.org/rcptt/release/${version}/ide/rcptt.ide-${version}-linux.gtk.x86_64.zip";
+    sha256 = "KofgdIHCmi0AeF7oQjp+kvkvuPX6+D7o2Z9U35qwJLA=";
+  };
+
+  buildInputs = with xorg; [
+    fontconfig freetype glib gsettings-desktop-schemas gtk3 jdk11 xorg.libX11
+    libXrender libXtst libsecret makeWrapper zlib unzip
+    rsync
+  ] ++ lib.optional (webkitgtk != null) webkitgtk;
```

Comment:
Please import `libX11` directly. Also, looks like rsync and unzip should be in `nativeBuildInputs`. 

Is there a need to use `jdk`? Maybe it would be sufficient to just use `jre`?

Reply:
Yes. A JRE is sufficient.